src/sql-parser/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 itertools = "0.9.0"
 lazy_static = "1.4.0"
 log = "0.4.13"
-ore = { path = "../ore" }
+ore = { path = "../ore", features = ["either"] }
```

Comment:
The either stuff appears to only be used in exactly one callsite. I propose we just remove Either entirely from the codebase.